plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -202,9 +221,10 @@ instance HasErrorCode CekUserError where
     errorCode CekEvaluationFailure {} = ErrorCode 37
     errorCode CekOutOfExError {}      = ErrorCode 36
 
-{- Note [Being generic over @term@ in 'CekM']
-We have a @term@-generic version of 'CekM' called 'CekCarryingM', which itself requires a
-@term@-generic version of 'CekEvaluationException' called 'CekEvaluationExceptionCarrying'.
+{- Note [Being generic over 'term' in errors]
+Our error for the CEK machine carries a term with it as a possible 'cause'. This is defined in
+terms of an underlying type that is generic over this 'term' type.
```

Comment:
But we are not generic over `term` in `CekEvaluationException` and we don't have `CekEvaluationExceptionCarrying` anymore?